dds/InfoRepo/DCPSInfoRepo.mpc

```diff
@@ -156,4 +156,9 @@ project(*Main): dcpsexe, iortable, bidir_giop, DCPSInfoRepo_bits, imr_client, sv
 
   TypeSupport_Files {
   }
+
+  verbatim(gnuace, postinstall) {
+"	@$(MKDIR) $(DESTDIR)$(INSTALL_PREFIX)/share/dds/bin"
```

Comment:
Why is this a `share` file instead of just in `bin`?

Reply:
It was a fix for PerlDDS not finding info repo unless I did this. I was going to look into it more, but it looks like I accidentally committed it. Do you want me to remove it?

Reply:
Yes, if we're going to start using PerlDDS with the installed tree, then PerlDDS will have to be adapted to that.  Once it is then it could be useful for more general testing than just CI.